runtimes/tests/org.jboss.tools.rsp.server.wildfly.test/src/main/java/org/jboss/tools/rsp/server/wildfly/test/beans/MockServerCreationUtilities.java

```diff
@@ -550,8 +583,13 @@ private static void createTmpDir() {
 	protected static File extractToTmpDir(String path) {
 		File pathFile = new File(MOCK_TMP_DIR, path);
 		if( !pathFile.exists()) {
-			ClassLoader classLoader = MockServerCreationUtilities.class.getClassLoader();
```

Comment:
Any reasons you're switching this? I'm not convinced that's the right answer...

Reply:
When running test from eclipse, the class is not on the class path and it does not find server mock folder... That was the reason, I believe that there could be better way, though... This came to me like straightforward and easy solution working for both cases (running from ide and from maven).